Libs/Optimize/ParticleSystem/ContourDomain.cpp

```diff
@@ -0,0 +1,348 @@
+#include "ContourDomain.h"
+#include <iostream>
+#include <fstream>
+#include <vtkDijkstraGraphGeodesicPath.h>
+#include <vtkDoubleArray.h>
+
+
+namespace itk{
```

Comment:
😭

Reply:
:( needed for the itkSmartPointer stuff

Reply:
Yeah, I know it's a subclass of ParticleDomain